src/Symfony/README.md

```diff
@@ -0,0 +1,25 @@
+# Async AWS Symfony Bundle
+
+![](https://github.com/async-aws/symfony-bundle/workflows/Tests/badge.svg?branch=master)
+![](https://github.com/async-aws/symfony-bundle/workflows/BC%20Check/badge.svg?branch=master)
+
+An small SymfonyBundle that helps with configuration and autowiring.
+
+## Install
+
+```cli
+composer require async-aws/symfony-bundle
+```
+
+## Configure
+
+```yaml
+async_aws:
+  default: # Will be merged with other configuration
+    config:
+      region: eu-central-1
+  sqs:
```

Comment:
There an issue with this config structure. I can't declare 2 clients with differents configurations. Imagine, I want to copy files from a bucket to another that requires differents credentials.

What's about:
```yaml
async_aws:
  default: 
    region: eu-central-1
  foo:
    region: us-east-1
```

That generate the following Binds (check if class Exists or not)
```phpp
\AsyncAws\S3\S3Client // region: eu-central-1
\AsyncAws\S3\S3Client $fooClient // region: us-east-1
```

Drawback would be, the user won't have nice exception telling "please install composer req async-aws/sqs", that could be fixed with (but I wonder if it whroth it):

```yaml
async_aws:
  default: 
    region: eu-central-1
    services: [s3, sqs]
```

Reply:
But you cannot autowire for those two S3 clients...

But sure. May I suggest to use the “foo” key as service. Ie. If you write async_aws.foo you need async_aws.foo.type: s3. 

You could do just:

async_aws.s3: ~

Reply:
> But you cannot autowire for those two S3 clients

What's about "Autowiring by type and name" https://symfony.com/blog/new-in-symfony-4-2-autowiring-by-type-and-name

When using:
```php
function __construct(S3Client $foo) // $foo because I defined a key `foo:` in config
{
}
```
Like Symfony FW does for Cache adapters and Lock.


> If you write async_aws.foo you need async_aws.foo.type: s3.

Thats could be optionnal: When type is not provided, we can provide a binding for all existing clients

Reply:
This is fixed. See Readme